readme.md

```diff
@@ -324,7 +324,7 @@ Yes! Details in [Lighthouse configuration](./docs/configuration.md).
 ### How does Lighthouse use network throttling, and how can I make it better?
 
 Good question. Network and CPU throttling are applied by default in a Lighthouse run. The network
-attempts to emulate slow 4G and the CPU is slowed down 4x from your machine's default speed. If you
+attempts to emulate slow 4G, and the CPU is slowed down 4x from your machine's default speed. If you
```

Comment:
I think I'm getting caught up on the `slow 4G` wording as well.  I'd add a little more context to it.
```
The network attempts to emulate a slow 4G network, and the CPU is slowed down 4x from 
the machine's default speed.
```